doc/articles/toc.yml

```diff
@@ -32,6 +32,8 @@
     - name: Troubleshoot
       topicHref: xref:Uno.UI.CommonIssues
       items:
+      - name: Sign in with Uno Platform
+        href: xref:Uno.GetStarted.Licensing
       - name: All Development Environments
```

Comment:
Feels odd for this to be in the Troubleshooting section of the docs. Maybe Gettting Started makes more sense?

Reply:
@kazo0 it was already discussed [here](https://teams.microsoft.com/l/message/19:787ad1de1af24252b9041bcbd4ec1ca3@thread.tacv2/1730994911318?groupId=f55cdec8-036b-4efa-8e1a-c4129ba9bf12&parentMessageId=1730902542981&tenantId=a297d6c0-b635-41a3-b1e3-558efe71e413)